tests/extensions/test_datacube.py

```diff
@@ -6,6 +6,7 @@
 
 import pystac
 import pystac.extensions.datacube as dc
+from pystac import Item
```

Comment:
Nit: do you prefer this syntax in tests? I keep waffling, but I know I don't like having both `import pystac` _and_ `from pystac import Item`